src/Method/Payment/Payment.php

```diff
@@ -85,6 +85,10 @@ class Payment extends ResultObject
     /** @type ThreeDS2AuthorizationInformation */
     private $threeDS2AuthorizationInformation;
 
+    /** @type string a descriptor to include in statement provided by a merchant. limit will vary based on merchant name
+        Maximum length 25 characters. */
```

Comment:
This says max length 25 chars, but in Create.php, the element is set to max 255

Reply:
Also, the name is misspelled. Should be $statementDescriptorSuffix. Same for getters/setters.